src/EnergyPlus/ZoneTempPredictorCorrector.hh

```diff
@@ -284,7 +284,8 @@ namespace ZoneTempPredictorCorrector {
                       Real64 &SumMCp,     // Zone sum of MassFlowRate*Cp
                       Real64 &SumMCpT,    // Zone sum of MassFlowRate*Cp*T
                       Real64 &SumSysMCp,  // Zone sum of air system MassFlowRate*Cp
-                      Real64 &SumSysMCpT  // Zone sum of air system MassFlowRate*Cp*T
+                      Real64 &SumSysMCpT,             // Zone sum of air system MassFlowRate*Cp*T
+                      bool const CorrectorFlag = true // Corrector call flag
```

Comment:
Using a default value is fine, but it's also OK to just require passing it in, that way any client code has to intentionally set the arg to true or false.  No big deal.